torchao/csrc/cuda/rowwise_scaled_linear_cutlass/rowwise_scaled_linear_cutlass.cuh

```diff
@@ -293,42 +304,59 @@ template<typename ElementA, typename ElementB, typename... Types>
 static void select_config(
     const at::Tensor& tensor_a, const at::Tensor& tensor_a_scale,
     const at::Tensor& tensor_b, const at::Tensor& tensor_b_scale,
-    const at::Tensor& tensor_c, at::Tensor& tensor_d) { 
+    const at::Tensor& tensor_c, at::Tensor& tensor_d) {
   const auto dprops = at::cuda::getCurrentDeviceProperties();
   const auto is_sm8x = dprops->major == 8;
```

Comment:
@alexsamardzic Should this be `>= 8` instead? The kernel can still run on sm90 and above, but probably not optimal.

Reply:
We can change (and rename the flag to `sm8x_or_greater`), but it should be validated on sm90.  I was thinking to keep it as is, and follow up with dedicated sm90 kernel as soon as there is a request.